.github/workflows/test_descriptive_graph_metrics.yml

```
@@ -16,13 +16,10 @@ jobs:
       with:
         example-location: ./cognee/tests/tasks/descriptive_metrics/networkx_metrics_test.py
       secrets:
-        LLM_MODEL: ${{ secrets.LLM_MODEL }}
-        LLM_ENDPOINT: ${{ secrets.LLM_ENDPOINT }}
         LLM_API_KEY: ${{ secrets.LLM_API_KEY }}
-        LLM_API_VERSION: ${{ secrets.LLM_API_VERSION }}
-        EMBEDDING_MODEL: ${{ secrets.EMBEDDING_MODEL }}
-        EMBEDDING_ENDPOINT: ${{ secrets.EMBEDDING_ENDPOINT }}
-        EMBEDDING_API_KEY: ${{ secrets.EMBEDDING_API_KEY }}
-        EMBEDDING_API_VERSION: ${{ secrets.EMBEDDING_API_VERSION }}
+        EMBEDDING_PROVIDER: "fastembed"
```

actionlint (1.7.4)

secret "EMBEDDING_PROVIDER" is not defined in "./.github/workflows/reusable_python_example.yml" reusable workflow. defined secrets are "GRAPHISTRY_PASSWORD", "GRAPHISTRY_USERNAME", "LLM_API_KEY", "OPENAI_API_KEY"

(workflow-call)